db/schema.rb

```diff
@@ -57,6 +57,7 @@
   end
 
   add_index "api_keys", ["token"], name: "api_keys_token_idx", unique: true, using: :btree
+  add_index "api_keys", ["token"], name: "index_api_keys_on_token", using: :btree
```

Comment:
Please, do not commit any changes to `db/schema.rb` which are not related to the migration you added.

Reply:
Sorry, I don't know why these changes were generated automatically.